pkg/codegen/internal/test/testdata/other-owned/dotnet/Other.Example.csproj

```diff
@@ -0,0 +1,62 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Authors>Pulumi Corp.</Authors>
```

Comment:
Maybe unrelated - but should this be configurable too?

Reply:
It should be configurable. It's not strictly related, but I can add that to this PR.

Reply:
I think this should be driven by the `publisher` field: https://www.pulumi.com/docs/guides/pulumi-packages/schema/#pulumi-package-schema